### PR TITLE
Add Abstract<primitive>Stacks.

### PR DESCRIPTION
**Pull request by kristeno**
Signed-off-by: Kristen O'Leary kristen.oleary@gs.com

### CODE REVIEW

File: eclipse-collections-code-generator/src/main/resources/impl/stack/abstractPrimitiveStack.stg [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+
+    protected void checkSizeLessThanCount(int count)
+    {
+        if (this.getDelegate().size() \< count)
+        {
+            throw new IllegalArgumentException("Count must be less than size: Count = " + count + " Size = " + this.getDelegate().size());
+        }
+    }
+
+
```
Comment (itohro) [inline eclipse-collections-code-generator/src/main/resources/impl/stack/abstractPrimitiveStack.stg]: Very minor, but please delete a new line here.

File: eclipse-collections-code-generator/src/main/resources/impl/stack/abstractPrimitiveStack.stg [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+    {
+        this.each(procedure);
+    }
+
+    public <name>Iterator <type>Iterator()
+    {
+        return new Unmodifiable<name>Iterator(this.getDelegate().asReversed().<type>Iterator());
+    }
+
+
```
Comment (itohro) [inline eclipse-collections-code-generator/src/main/resources/impl/stack/abstractPrimitiveStack.stg]: Same here, delete a new line here.

**Comment by kristeno** [comment]
Thanks! Will make the changes and rebase.

**Comment by itohro** [comment]
Thanks for the change. Used "rebase and merge" functionality in GitHub so no worries about the rebase.

**Pull request merged**